src/pudl/metadata/resources/ferceqr.py

```diff
@@ -0,0 +1,136 @@
+"""Table definitions for the FERC EQR data group."""
+
+from typing import Any
+
+TABLE_DESCRIPTIONS = {
+    "ident": {
+        "additional_summary_text": "placeholder",
+        "additional_source_text": "placeholder",
+        "usage_warnings": [],
+    },
+    "contracts": {
+        "additional_summary_text": "placeholder",
+        "additional_source_text": "placeholder",
+        "usage_warnings": [],
+    },
+    "transactions": {
+        "additional_summary_text": "placeholder",
+        "additional_source_text": "placeholder",
+        "usage_warnings": [],
+    },
+    "index_pub": {
+        "additional_summary_text": "placeholder",
+        "additional_source_text": "placeholder",
+        "usage_warnings": [],
+    },
+}
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
```

Comment:
Even though we're not defining the database schema for these tables, we should explicitly list the primary key if there is one.

Reply:
Added a todo to add primary keys. I want to do some validation that the primary keys are actually unique before adding them though.